api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -7,16 +7,23 @@ class Jpi::V1::Admin::OrganizationsController < Jpi::V1::Admin::BaseResourceCont
                        :geo_country_code, :geo_state_code, :geo_city,
                        :geo_tz, :geo_currency, :metadata, :industry, :size,
                        :financial_year_end_month, :credit_card,
-                       :financial_metrics, :created_at, :external_id]
+                       :financial_metrics, :created_at, :external_id, :belong_to_sub_tenant, :belong_to_account_manager]
+
     # GET /mnoe/jpi/v1/admin/organizations
     def index
       if params[:terms]
         # Search mode
         @organizations = []
         JSON.parse(params[:terms]).map do |t|
-          @organizations = @organizations | MnoEnterprise::Organization.with_params(_metadata: { act_as_manager: current_user.id })
-                                              .select(INCLUDED_FIELDS)
-                                              .where(Hash[*t])
+
+          query = MnoEnterprise::Organization
+                    .apply_query_params(params.except(:terms))
+                    .select(INCLUDED_FIELDS)
+                    .with_params(_metadata: { act_as_manager: current_user.id })
+                    .where(Hash[*t])
+          query = query.with_params(sub_tenant_id: params[:sub_tenant_id]) if params[:sub_tenant_id]
```

Comment:
this is to retreive the belong_to_sub_tenant and belong_to_account_manager property.